app/controllers/api/v1/errors_controller.rb

```diff
@@ -6,6 +6,17 @@ def index
   end
 
   def create
+    @subscriber = Subscriber.where("email = ? and website_id = ?", params['email'], @current_site.id)
```

Comment:
nu mai folosi `@` decat daca vrei sa folosesti `@subscriber` in view adica in .html .json files. Altfel nu se justifica sa folosesti @.